src/library_webgl.js

```diff
@@ -4194,11 +4194,10 @@ function createGLPassthroughFunctions(lib, funcs) {
     const returnStub = '(function(' + args + ') { return GLctx[\'NAME\'](' + args + ') })';
     const sigEnd = range(num).map(() => 'i').join('');
     names.split(' ').forEach((name) => {
```

Comment:
Maybe the split could be removed here, and directly have the above thing contain arrays, e.g. `[0, ['finish', 'flush']]`?

(Could be a separate improvement, or leave out, lgtm either way)